nan_maybe_pre_43_inl.h

```diff
@@ -16,9 +16,10 @@ class MaybeLocal {
 
   template<typename S>
 # if NODE_MODULE_VERSION >= NODE_0_12_MODULE_VERSION
-  inline MaybeLocal(v8::Local<S> that) : val_(that) {}
+  inline MaybeLocal(v8::Local<S> that) :  // NOLINT(runtime/explicit)
+      val_(that) {}
 # else
-  inline MaybeLocal(v8::Local<S> that) :
+  inline MaybeLocal(v8::Local<S> that) :  // NOLINT(runtime/explicit)
       val_(*reinterpret_cast<v8::Local<T>*>(&that)) {}
```

Comment:
The conversion constructor doesn't allocate a HandleScope slot.  A `TYPE_CHECK` is fine too, though.